applicationinsights-core-js.ts

```diff
@@ -7,4 +7,6 @@ export { INotificationListener } from "./JavaScriptSDK.Interfaces/INotificationL
 export { EventsDiscardedReason } from "./JavaScriptSDK.Enums/EventsDiscardedReason";
 export { AppInsightsCore } from "./JavaScriptSDK/AppInsightsCore";

```

Comment:
can u also update the package.json versions in amd\package.json and cjs\package.json. Thanks!